### PR TITLE
RDMA: Support IMB-EXT & IMB-IO in Intel MPI Benchmark

### PR DESCRIPTION
**Pull request by juhlee-microsoft**
Added IMB-EXT and IMB-IO tests from Intel MPI Benchmark.

### CODE REVIEW

File: Testscripts/Linux/SetupRDMA.sh [hunk 1]
```diff
@@ -390,9 +390,18 @@ function Main() {
...
-	LogMsg "Intel Benchmark test installation completed"
+
+	# install P2P test
+	LogMsg "Change directory to P2P"
+	cd P2P
+	LogMsg "Renaming from mpiicc to mpicc in Makefile"
+	sed -i -e 's/CC=mpiicc/CC=mpicc/g' Makefile
+	LogMsg "Building P2P binary"
+	make -j $(nproc)
+	LogMsg "Completed P2P2 binary compilation"
```
Comment (ader1990) [inline Testscripts/Linux/SetupRDMA.sh]: P2P2 or P2P?

Comment (juhlee-microsoft) [reply Testscripts/Linux/SetupRDMA.sh]: fixed in 9c875c4e590774f8a8505f5b9e1f86d9236e4be4

**Comment by juhlee-microsoft** [comment]
#153 must be merged to master first

**Comment by juhlee-microsoft** [comment]
need to work more.

**Pull request closed without merging**